vcpkg/concepts/system-package-managers.md

```diff
@@ -0,0 +1,23 @@
+---
+title: System package managers
+description: Concepts on system package managers
+author: data-queue
+ms.author: danshaw2
+ms.date: 01/10/2024
+ms.prod: vcpkg
+ms.topic: concept-article
+---
+
+# System package managers
+
+C++ libraries are available in system package managers, such as apt, and vcpkg. How you choose to acquire your libraries depends on your specific usecase.
+
+## Pros of using dependencies from system package manager
```

Comment:
It would be good to link to https://devblogs.microsoft.com/cppblog/using-system-package-manager-dependencies-with-vcpkg/ from here (or copy that blog post effectively to `learn.`) so that readers have any idea what it means to do this.